packages/core/src/Account.ts

```diff
@@ -230,16 +229,6 @@ export class Account extends TypedEventEmitter<Events> {
     return storeEngine.updateOrCreate(AUTH_TABLE_NAME, AUTH_COOKIE_KEY, entity);
   }
 
-  private async getE2EIStatus() {
-    const features = await this.apiClient.api.teams.feature.getAllFeatures();
-    const clientCanUseE2EI = this.coreCryptoConfig?.mls?.useE2EI;
-    const teamCanUseE2EI = features[FEATURE_KEY.MLSE2EID]?.status === FeatureStatus.ENABLED;
-
-    return {
-      isFeatureEnabled: clientCanUseE2EI && teamCanUseE2EI,
-    };
-  }
```

Comment:
It's the responsibility of the consumer to get the config. The core is too low level to parse those kind of config